boot/debug.go

```diff
@@ -67,6 +68,8 @@ func DumpBootVars(w io.Writer, dir string, uc20 bool) error {
 			"snapd_recovery_kernel",
 			"snap_kernel",
 			"kernel_status",
+			"recovery_system_status",
```

Comment:
hmm shouldn't we also support snap_try_kernel here too? (not that your PR is to blame for this of course, git blame says it's my fault :-O)

Reply:
I'll add it

Reply:
thanks!